capstone/src/test/java/com/google/sps/LessonSerializationTest.java

```diff
@@ -18,6 +18,8 @@
 import java.io.IOException;
 import com.google.appengine.api.datastore.Entity;
 
+// Update Tests in next PR
+
```

Comment:
The previous PR has the updated tests. You could take this out when you merge up

Reply:
Got it